packages/x-components/src/design-system/components/suggestion-group/default.scss

```diff
@@ -29,6 +28,18 @@
   gap: var(--x-size-gap-suggestion-group-default);
   @include safari-gap(var(--x-size-gap-suggestion-group-default));
 
+  // border
+  border: none;
+  border-block-start-width: var(--x-size-border-width-top-suggestion-group-default);
+  border-block-end-width: var(--x-size-border-width-bottom-suggestion-group-default);
+  border-inline-start-width: var(--x-size-border-width-left-suggestion-group-default);
+  border-inline-end-width: var(--x-size-border-width-right-suggestion-group-default);
+  border-radius: var(--x-size-border-radius-top-left-suggestion-group-default)
+  var(--x-size-border-radius-top-right-suggestion-group-default)
+  var(--x-size-border-radius-bottom-right-suggestion-group-default)
+  var(--x-size-border-radius-bottom-left-suggestion-group-default);
+
```

Comment:
I'm trying to understand the decisions 😅, why do we need to add these properties? If the suggestion/suggestion-group don't specify a border, but the tag does, it will also work and it will be less verbose and less coupled

Reply:
maybe we want a different border radius if it is a suggestion. I mean, we will have a suggestion that will be a tag and maybe we want a different border-radius in that case. Makes sense?

Reply:
There are no designs for this? We could add endless variables but as far as I know we where just adding the ones that were needed in the design.